backend/services/implementations/testSessionService.ts

```diff
@@ -62,8 +62,11 @@ class TestSessionService implements ITestSessionService {
       teacherDTO = await this.userService.getUserById(testSession.teacher);
       schoolDTO = await this.schoolService.getSchoolById(testSession.school);
 
-      newTestSession = await MgTestSession.create(testSession);
+      if (testSession.startDate > testSession.endDate) {
```

Comment:
should this be something controlled by the FE? another check would maybe for if endDate < currentDate

if FE handles, then we should update docs to mention this guarantee

Reply:
i think in general its best practice to do validation on both the frontend and backend? i don't think super critical to have for our use case, but also why not 🤷‍♀️

https://www.quora.com/Should-I-do-input-validation-on-the-front-end-or-back-end
https://stackoverflow.com/questions/37492271/should-i-handle-html-form-validation-on-front-or-backend

but good point for the `end < current`, I will add that!

Reply:
wait `end > current` right? :D